src/main/java/com/animalfinder/dto/BreedDTO.java

```diff
@@ -2,15 +2,15 @@
 
 public class BreedDTO {
 
-	private int BreedID;

```

Comment:
Neither, actually.  Variable names in Java should be lowerCamelCase, not UpperCamelCase.